### DISCUSSION TITLE
how combine with esbuild-coffeescript ?

### DISCUSSION THREAD

**Discussion by MatrixFr** [EMH333/esbuild-svelte#142] [Q&A]
I try to use coffeescript in svelte on my rails app and i don't understand how merge both :

```
#!/usr/bin/env node

const watch              = process.argv.includes("--watch");
const esbuild            = require('esbuild')
const coffeeScriptPlugin = require('esbuild-coffeescript');
const esbuildSvelte      = require('esbuild-svelte');
const sveltePreprocess   = require('svelte-preprocess');

esbuild
  .build({
    entryPoints: ["app/javascript/svelte_src.js"],
    bundle: true,
    outfile: "app/assets/builds/svelte.js",
    plugins: [
      esbuildSvelte({
        preprocess: sveltePreprocess(/*{coffeescript: { bare: true }}*/),
      }),
      coffeeScriptPlugin({bare: true}),
    ],
    logLevel: "debug",
    watch: watch
  })
  .catch(() => process.exit(1));

```

I tried add plugin at the different place, but not working...

I find https://github.com/svelte-add/coffeescript but with rails we use esbuild and there is not sveltekit, so I cant use this... how can I do?

**Comment by EMH333** [comment]
Are you able to get the build to work without the coffeescript preprocessing? Since it looks like you are using `svelte-preprocess` I'm not expecting that side of things to be the issue
